solution-base/build.sh

```diff
@@ -32,9 +32,7 @@ SOLUTION_ENV='SOLUTION_ENV'
 export KUBEDB_OPERATOR_TAG=$(grep kubedb/operator: deps.txt | awk -F ':' '{print $2}')
 export KUBEDB_NAMESPACE=${SOLUTION_ENV}
 export KUBEDB_SERVICE_ACCOUNT=kubedb-operator
-export KUBEDB_IMAGE_NAME=operator
```

Comment:
Was it intended to commit those changes? It does not seem related to documentation, hence asking.

Reply:
looks like one of my commits from solution work, not sure how it ended up here

Reply:
I am confident that this was the result of some Git buffoonery on my part. I'll see if I can roll it back.

Reply:
I just copy/pasted this back to "before", figuring it would be easier and more predictable than my trying to track down and modify the bad commit via Git magic. If this is a problem, let me know, and I'll work on a cleverer response.

Reply:
Hoo boy. Scrolling down, I see this is worse than I thought. I'll rebase and hope Ronnie's mods are taken care of.

Reply:
I think that took care of it.